composer.json

```diff
@@ -1,15 +1,41 @@
 {
-  "name":"ec-europa/search_api_europa_search",
-  "type":"drupal-module",
-  "description":"Provides an Europa Search backend for the Search API module.",
+  "name": "ec-europa/search_api_europa_search",
+  "type": "drupal-module",
+  "description": "Provides an Europa Search backend for the Search API module.",
+  "minimum-stability": "dev",
```

Comment:
If you want used one dev requirement, is recommended to use stability flags.
Example:
```
"minimum-stability": "stable",
"require": {
  ...
  "ec-europa/oe-europa-search-client": "dev-nept-1558-issue17@dev"
},
```

See more information here: https://igor.io/2013/02/07/composer-stability-flags.html